README.md

```
@@ -1,70 +1,85 @@
+# Start the bot
+yarn start
+
+# Run tests
+yarn test
+```
+
+## Architecture
 
-  # Add the template remote
-  git remote add template https://github.com/ubiquity/ts-template
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)